convert2rhel/pkghandler.py

```diff
@@ -1057,3 +1057,21 @@ def _parse_pkg_with_dnf(pkg):
 
     pkg_ver_components = (name, epoch, version, release, arch)
     return pkg_ver_components
+
+
+def get_highest_package_version(pkgs):
+    """
+    Get the highest version from the provided list of packages.
+    :param pkgs: A tuple containing the name of the package list (as a string) and the list of package versions (as a list of strings).
+    :type pkgs: tuple(str,list[str])
+    :return: Return a single package with the highest version.
+    :rtype: str
+
+    :raises ValueError: If the list is empty, raise a ValueError.
```

Comment:
Atm this is untrue as it doesn't reraise the exception. If the list is empty it returns None atm

Reply:
Right, I forgot to put it back :sweat_smile: 
Thanks!